meson.build

```diff
@@ -343,8 +343,13 @@ if host_machine.system() == 'freebsd'
   add_project_link_arguments('-Wl,--allow-shlib-undefined', language: 'c')
 endif
 
-lrt = []
+lrt_gettime = []
```

Comment:
why this difference? isn't it the same?

Reply:
The purpose is different, the gettime one is used only for `clock_gettime()` in `librz/util/` and `lrt` is used for `shm_open()`.

Reply:
I don't think it matters. The library is the same, so it doesn't really matter.
Actually, below you could do:

```
if host_machine.system() != 'openbsd' and lrt != []
```
to avoid the double looking up for `rt`. I'm pretty sure there is no need to have two names for the same lib.

Reply:
Good idea, done.